recipe/conda_build_config.yaml

```diff
@@ -181,8 +181,6 @@ pin_run_as_build:
     max_pin: x.x
   gdk_pixbuf:
     max_pin: x.x.x
-  geos:
-    max_pin: x.x.x
```

Comment:
@isuruf if we we have `run_exports` we can remove this, right?

Reply:
If all builds of version `3.7.2` have a `run_exports`, then yes.

Reply:
Yep. We are good. We have `run_exports` there since `v3.6.2`.